jbehave-core/src/main/java/org/jbehave/core/embedder/StoryControls.java

```diff
@@ -87,6 +96,10 @@ public boolean skipStoryIfGivenStoryFailed() {
         return currentStoryControls().skipStoryIfGivenStoryFailed;
     }
 
+    public DecimalFormat storyIndexFormat() {
+        return Optional.ofNullable(currentStoryControls().storyIndexFormat).orElseGet(() -> new DecimalFormat(" [0]"));
```

Comment:
why not use default init at the field level as it's done with all fields of StoryControls class?

Reply:
`DecimalFormat` is not thread-safe